web/src/components/people/TrainingForm.tsx

```diff
@@ -0,0 +1,136 @@
+import { FormEvent, useState } from "react";
+import styles from "./TrainingForm.module.css";
+import Button from "../ui/Button";
+import { postPeopleById, TrainingKind, TrainingRecord } from "@/openapi";
+import dynamic from "next/dynamic";
+import { AlertType } from "uikit-react-public/dist/components/Alert/Alert";
+import AdminDialog from "./AdminDialog";
+
+const Alert = dynamic(() => import("uikit-react-public").then((mod) => mod.Alert), {
+  ssr: false,
+});
+const AlertMessage = dynamic(() => import("uikit-react-public").then((mod) => mod.Alert.Message), {
+  ssr: false,
+});
+
+const TrainingKindOptions = {
+  //  is there a better way of doing this? Won't let me use type as a value
+  nhsd: "training_kind_nhsd",
+};
```

Comment:
@t-young31 is there a backend implementation that fetches all training kinds? Is it worth building this functionality?

Reply:
not yet because there's only one training kind so far – I vote we add it when there is